nixos/extractors/services.nix

```diff
@@ -36,6 +39,29 @@ in {
           details.listen = mkIf (address != null && port != null) {text = "${address}:${toString port}";};
         };
 
+      authelia = let
+        instances =
+          filterAttrs (_: v: v.enable) config.services.authelia.instances;
+      in
+        mkIf (instances != {}) {
+          name = "Authelia";
+          icon = "services.authelia";
+          details.listen.text =
+            concatStringsSep " " (mapAttrsToList (n: v: "${n}: ${v.settings.server.host}:${toString v.settings.server.port}")
+              instances);
+        };
+
+      blocky = mkIf config.services.blocky.enable {
+        name = "Blocky";
+        icon = "services.blocky";
+        info =
```

Comment:
Same thing here, I think it would increase readability to use separate detail values here, i.e. `details."listen.${n}" = ${v}`. The `info` property should usually show user-facing URLs, so users of block will probably set that themselves.